sdk/internal/recording/recording.go

```diff
@@ -473,11 +473,21 @@ func defaultOptions() *RecordingOptions {
 	}
 }
 
-func (r RecordingOptions) hostScheme() string {
+func (r RecordingOptions) Host() string {
```

Comment:
In `http://localhost:5000`, "http" is the scheme and "localhost" the host ("localhost:5000" is the authority)

Reply:
Getting the proxy URL ended up being a lot of repeated code, I think it's better to have a method `ProxyURL` that returns the full URL to the proxy instead of separate URLs for the Host and Scheme.